src/Files.App/UserControls/AddressToolbar.xaml

```diff
@@ -349,6 +349,20 @@
 					ToolTipService.ToolTip="{x:Bind Commands.RefreshItems.LabelWithHotKey, Mode=OneWay}">
 					<FontIcon FontSize="14" Glyph="{x:Bind Commands.RefreshItems.Glyph.BaseGlyph}" />
 				</Button>
+
+				<Button
+					x:Name="HomeButton"
+					x:FieldModifier="public"
+					x:Load="{x:Bind ViewModel.ShowHomeButton, Mode=OneWay}"
```

Comment:
How about using IsExecutable property of NavigateHomeAction and put the user setting condition on it?

Reply:
The action should still be useable even if the home button is hidden.

Reply:
Fair enough.